experiment-rollout.js

```diff
@@ -44,29 +44,25 @@ const data = {
         experimentType: 1,
         rolloutType: 1,
         priority: [
-            {
-                status: 1,
-                name: 'Verified bots (Server count wins name conflicts, other ones will get their discriminator appended)'
-            },
             {
                 status: 2,
                 name: 'Brands'
             },
             {
                 status: 2,
-                name: 'Discord staff'
+                name: 'Discord Staff'
             },
             {
-                status: 1,
-                name: 'Partnered and verified server owners (Includes old ones + some content creators)'
+                status: 2,
+                name: 'Partnered, subscription-enabled (100USD/mo+), and verified server owners (includes some old partners + some content creators)'
             },
             {
                 status: 1,
-                name: 'Moderator programs alumnis'
+                name: 'Verified bots (Server count wins name conflicts, other ones will get their discriminator appended)'
             },
             {
                 status: 0,
-                name: 'HypeSquad event owners'
+                name: 'Top (likely 100, by server count) bot developers'
```

Comment:
Also where did you get this info?

Reply:
While this is extremely likely considering the risk mitigation approach to rolling out Pomelo, it has only been hinted by Discord Staff. I can relabel it as speculation if you'd like.

Reply:
Yeah i'd like it